### PR TITLE
Better Fix for XSS

### PR DESCRIPTION
**Pull request by rohitdatta**
**Fixes #174.**

**Changes proposed in this pull request:**

* Discovered in staging that XSS fix didn't work, so haven't deployed into prod

**Have you made sure to add:**
- [X] Tests
- [ ] Documentation

**Any Additional Information**

### CODE REVIEW

File: formspree/utils.py [hunk 1]
```diff
@@ -11,8 +11,8 @@
 IS_VALID_EMAIL = lambda x: re.match(r"[^@]+@[^@]+\.[^@]+", x)
 
 def valid_url(url):
-    parsed = urlparse.urlparse(url)
-    return len(parsed.scheme) > 0 and len(parsed.netloc) > 0
+    parsed = urlparse.urlparse('http://' + url)
+    return len(parsed.scheme) > 0 and len(parsed.netloc) > 0 and not 'javascript:' in url
```
Comment (fiatjaf) [inline formspree/utils.py]: If you're prepending an `http://` to the URL it's 100% guaranteed that it will have a `scheme`, right?

Comment (fiatjaf) [reply formspree/utils.py]: Actually, I don't understand what is the problem with the previous solution.

If the next-URL was generated by [`next_url()`](https://github.com/formspree/formspree/blob/6320760a45f842f43238cde5e439606736e3b4c4/formspree/utils.py#L96-L116), as it should in all normal cases, it must have a `scheme` and a `netloc` already, so there's no need to prepend that `http://` there.

What am I missing?

Comment (rohitdatta) [reply formspree/utils.py]: Yes you're right. I'll fix this accordingly.

**Comment by fiatjaf** [comment]
Looks good to me.

**Pull request merged**